src/e2e-tests/features/autelis.feature

```diff
@@ -0,0 +1,96 @@
+        Then Autelis <cleanerButton> button is disabled
+        When User clicks on <poolHeatButton> button to change state to On on Autelis page
+        Then Autelis <poolHeatButton> button is enabled
+        When User clicks on <poolHeatButton> button to change state to Off on Autelis page
+        Then Autelis <poolHeatButton> button is disabled
+        When User clicks on <poolLightButton> button to change state to On on Autelis page
+        Then Autelis <poolLightButton> button is enabled
+        When User clicks on <poolLightButton> button to change state to Off on Autelis page
+        Then Autelis <poolLightButton> button is disabled
+        When User clicks on <waterfallButton> button to change state to On on Autelis page
+        Then Autelis <waterfallButton> button is enabled
+        When User clicks on <waterfallButton> button to change state to Off on Autelis page
+        Then Autelis <waterfallButton> button is disabled
+        Examples:
+            | solarBtn | cleanerButton | poolHeatButton | poolLightButton | waterfallButton |
+            | Solar    | Cleaner       | Pool Heat      | Pool Light      | Waterfall       |
+
+
+    @S1 @automated
+    Scenario Outline: Check Spa Heat, Jets, Spa Light, Blower options 'On' and 'Off'
+        Given User loads the RoboDomo web app
+        When User clicks Autelis menu button
+        Then User is redirected to Autelis page
+        Then Autelis tab shall be selected
+        When User clicks on Autelis tab
+        When User clicks on <spaHeatButton> button to change state to On on Autelis page
+        Then Autelis <spaHeatButton> button is enabled
+        When User clicks on <spaHeatButton> button to change state to Off on Autelis page
+        Then Autelis <spaHeatButton> button is disabled
+        When User clicks on <jetsButton> button to change state to On on Autelis page
+        Then Autelis <jetsButton> button is enabled
+        When User clicks on <jetsButton> button to change state to Off on Autelis page
+        Then Autelis <jetsButton> button is disabled
+        When User clicks on <spaLightButton> button to change state to On on Autelis page
+        Then Autelis <spaLightButton> button is enabled
+        When User clicks on <spaLightButton> button to change state to Off on Autelis page
+        Then Autelis <spaLightButton> button is disabled
+        When User clicks on <blowerButton> button to change state to On on Autelis page
+        Then Autelis <blowerButton> button is enabled
+        When User clicks on <blowerButton> button to change state to Off on Autelis page
+        Then Autelis <blowerButton> button is disabled
+        Examples:
+            | spaHeatButton | jetsButton | spaLightButton | blowerButton |
+            | Spa Heat      | Jets       | Spa Light      | Blower       |
+
+    @S1 @automated
+    Scenario: Check Pool/Spa heat adjustments to lower and higher values
```

Comment:
Same for this Scenario as the above. Please decide on a common or independent approach.  Pool and Spa are different areas / devices / sensors, Maybe independently is better

Reply:
Updated